src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/ProvidersController.cs

```diff
@@ -130,70 +121,37 @@ public void Head(long ukprn)
         [ExceptionHandling]
         public ApprenticeshipTrainingSummary GetActiveApprenticeshipTrainingByProvider(long ukprn)
+        {
+            return GetActiveApprenticeshipTrainingByProvider(ukprn, 1);
+        }
+
+        /// <summary>
+        /// Get list of active apprenticeships for a given provider
+        /// </summary>
+        /// <param name="ukprn">unique id</param>
+        /// <param name="page">number of page for which results are returned (default 1)</param>
+        /// <returns>A list of active apprenticeships sorted by name alphabetically, then type, then level</returns>
+        [SwaggerResponse(HttpStatusCode.OK, "OK", typeof(IEnumerable<ApprenticeshipTraining>))]
+        [SwaggerResponse(HttpStatusCode.InternalServerError)]
+        [SwaggerResponse(HttpStatusCode.BadRequest, BadUkprnMessage)]
+        [Route("providers/{ukprn}/active-apprenticeship-training/{page}", Name = "GetActiveApprenticeshipsByProviderByPage")]
+        [ExceptionHandling]
+        public ApprenticeshipTrainingSummary GetActiveApprenticeshipTrainingByProvider(long ukprn, int page)
         {
             if (ukprn.ToString().Length != 8)
             {
                 throw HttpResponseFactory.RaiseException(HttpStatusCode.BadRequest, BadUkprnMessage);
             }
 
-            var apprenticeshipTrainingSummary = new ApprenticeshipTrainingSummary {Ukprn = ukprn};
-
-            var apprenticeships = new List<ApprenticeshipTraining>();
-
-            apprenticeships.AddRange(GetActiveStandardsForUkprn(ukprn));
-            apprenticeships.AddRange(GetActiveFrameworksForUkprn(ukprn));
-
-            var take = _applicationSettings.TakeMaximum;
-
-            apprenticeshipTrainingSummary.Count = apprenticeships.Count;
-            apprenticeshipTrainingSummary.ApprenticeshipTrainingItems
-                                = apprenticeships.OrderBy(x => x.Name)
-                                    .ThenBy(x => x.Level)
-                                    .Take(take);
-
-            return apprenticeshipTrainingSummary;
-        }
-
-        private IEnumerable<ApprenticeshipTraining> GetActiveFrameworksForUkprn(long ukprn)
-        {
-            var frameworks = _getProviders.GetFrameworksByProviderUkprn(ukprn);
-
-            return frameworks
-                .Where(x => _activeApprenticeshipChecker.CheckActiveFramework(x.FrameworkId, x.EffectiveFrom, x.EffectiveTo))
-                .Select(framework => new ApprenticeshipTraining
-                {
-                    Name = framework.PathwayName,
-                    Level = framework.Level,
-                    Type = ApprenticeshipTrainingType.Framework.ToString(),
-                    TrainingType = ApprenticeshipTrainingType.Framework,
-                    Identifier = framework.FrameworkId
-                })
-                .ToList();
-        }
-
-        private IEnumerable<ApprenticeshipTraining> GetActiveStandardsForUkprn(long ukprn)
```

Comment:
Why is this removed?

Reply:
I moved the processing logic that used this from the controller to the repository, so it is no longer used in the controller